libcaf_net/caf/net/socket_manager.hpp

```diff
@@ -44,17 +44,17 @@ class CAF_NET_EXPORT socket_manager : public detail::atomic_ref_counted,
   /// Returns the handle for the managed socket.
   virtual socket handle() const = 0;
 
-  /// @private
-  virtual void handle(socket new_handle) = 0;
-
-  /// Returns a reference to the hosting @ref actor_system instance.
-  virtual actor_system& system() noexcept = 0;
-
   /// Returns the owning @ref multiplexer instance.
-  virtual multiplexer& mpx() noexcept = 0;
+  virtual multiplexer& mpx() {
```

Comment:
Make up your mind. Either it's virtual and belongs to the `.cpp` file or it is inline and can be implemented here.

Reply:
Sorry, I meant to remove `virtual`.